libraries/provision/ansible/playbooks/start-sgload.yml

```diff
@@ -0,0 +1,82 @@
+---
+- hosts: load_generators
+  any_errors_fatal: true
+
+  # Kill gateload
+  tasks:
+  - name: kill previous sgload screen
+    shell: screen -X -S sgload kill
+    ignore_errors: yes
+  - name: kill previous sgload_expvars screen
+    shell: screen -X -S sgload_expvars kill
+    ignore_errors: yes
+
+- hosts: sync_gateways
+  any_errors_fatal: true
+  sudo: true
+
+  # Copy machine stat collection script to sync_gateways
```

Comment:
Do we still need these if we are using telegraf plugins for some of these things now?